src/nvim/api/vim.c

```diff
@@ -704,12 +756,21 @@ void nvim_unsubscribe(uint64_t channel_id, String event)
   channel_unsubscribe(channel_id, e);
 }
 
+/// Translate to integer if \p name is an hex value (e.g. #XXXXXX),
+/// else look into \p color_name_table to translate a color name to  its
```

Comment:
color_name_table sounds like an internal datastructure. Also the descriptions of `nvim_get_color_map` sounds like they use two different color tables.